themes/classic/templates/catalog/_partials/product-discounts.tpl

```diff
@@ -0,0 +1,23 @@
+<section class="product-discounts">
+  {if $product.quantity_discounts}
+    <h3 class="h6 product-discounts-title">{l s='Volume discounts' d='Shop.Theme.Catalog'}</h3>
+    <table class="table-product-discounts">
+      <thead>
+      <tr>
+        <th>{l s='Quantity' d='Shop.Theme.Catalog'}</th>
+        <th>{if $display_discount_price}{l s='Price' d='Shop.Theme.Catalog'}{else}{l s='Discount' d='Shop.Theme.Catalog'}{/if}</th>
```

Comment:
Maybe we should move this to the ProductPresenter, so we can have something like `$product.quantity_discount_type` or `$product.quantity_discount_label` and avoid this translation and if/else in the view